### ISSUE TITLE
How to change the background? No clues!

### ISSUE THREAD

**Issue by miamichix** [pH7Software/pH7-Social-Dating-CMS#278]
I cannot change the background? Why is this so complicated? Why is there template files but barely no data in it from admin panel to edit? There is no forum for users to get any idea on customization or anything. There is no responses to my emails to hello@ whatever email address. I did the patron payment and got confused on why that was needed when there is no support anywhere on the internet! So I am cancelling that asap. 

No information on how to buy the pro version. I installed via Cpanel.

**Comment by pH-7** [comment]
Hi @miamichix 

You can change the colour background simply through your admin panel -> Setting -> General Settings -> Design (color) tag.

Hope this helps! 😊

**Comment by miamichix** [comment]
If someone else have better knowledge than this person with limited comment, please explain how this script can be customize. How to un-closed this? His answer is as not the solution

**Comment by pH-7** [comment]
Hi @miamichix 

You didn't specify you wanted to change the background to an "image". How could I know that? Next time, please be more specific in order to avoid misunderstandings 😊

You can do so with CSS code thanks to the CSS Overrider integrated (Admin Panel -> Setting -> CSS Style).

Try to add something like below
```css
background-image: url('http://YOUR-WEBSITE.com/PATH_TO_IMAGE/MY_IMAGE.png') !important;
background-size: cover;
````

Enjoy your weekend!

**Comment by miamichix** [comment]
So i cannot do this: URL('../body_bg.png') or URL('./body_bg.png') instead? Right now, it is https: now and may be http: between server changes or whatnot. I wouldn't want to keep changing the whole domain URL between changes.

Sent from my MetroPCS 4G LTE Android device

On Aug 25, 2018 4:04 PM, ♚ PH⑦ Soria ♛ <notifications@github.com> wrote:

Hi @miamichix<https://github.com/miamichix>

You didn't specify you wanted to change the background to an "image". How could I know that? Next time, please be more specific in order to avoid misunderstandings 😊

You can do so with CSS code thanks to the CSS Overrider integrated (Admin Panel -> Setting -> CSS Style).

Try to add something like below

background-image: url('http://YOUR-WEBSITE.com/PATH_TO_IMAGE/MY_IMAGE.png') !important;
background-size: cover;

Enjoy your weekend!

**Comment by miamichix** [comment]
Your answer did not work as I started to implement it today. I went to the w3 site and it didn't work there. What's the issue? 

So I followed the correct format with "URL" instead 'URL' and it works in w3school test box.

So just to be clear and no "misunderstanding", I used:

Background-image: #000000 URL("https://www.domain.com/body_bg.jpg"); in the w3 testbox. It works in w3 website.  I copied and pasted the same CSS code to (settings - css) box on my site and it did NOT work. 

There is a bug with processing css custom box.
